flict/flictlib/license.py

```diff
@@ -2,10 +2,22 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from flict.flictlib.alias import Alias
 from flict.flictlib.license_parser import LicenseParserFactory
 from flict.flictlib.return_codes import FlictError, ReturnCodes
 
+from flame.license_db import FossLicenses # noqa: I900
+
+fl = None
+
+def compatible_license(license_expr, update_dual=True):
+    global fl
```

Comment:
Instead of this global singleton pattern I would propose to use a local pattern instead
```
if not hasattr(compatible_license, "fl"):
        compatible_license.fl = FossLicenses()
```